client/ayon_fusion/api/plugin.py

```diff
@@ -271,6 +271,13 @@ def _get_reviewable_bool(self):
             label="Review",
         )
 
+    def _get_thumbnail_only_bool(self):
+        return BoolDef(
+            "need_thumbnail",
+            default=False,
+            label="Include thumbnail only",
+        )
```

Comment:
I think the wording on this is weird.

Include it only? Compared to what? if set to only include it, what does it actually exclude?

Anyway, I also don't think this should be an artist toggle at all. I think the real problem is that disabling "Review" apparently disables the generation of Thumbnails.

I'd actually opt for that EITHER:
- It is correct that something that's not reviewable does not get a thumbnail (because how do we know that it could get one?)
- Or, always enforce generation of thumbnail when we can (regardless of whether something is review or not). This way, at the very least we'd get thumbnails as often as possible. Or is there a use case to also want to explicitly disable generation of a thumbnail?

@philippe-ynput @mkolar